src/librustc/README.md

```diff
@@ -1,204 +0,0 @@
-An informal guide to reading and working on the rustc compiler.
```

Comment:
Can you replace this with a README -- or maybe doc-comment on the crate -- that says to see the rustc guide? I know it's in other places, but I want to make it as visible as possible.